### PR TITLE
Formatting MessageHandler

### PR DESCRIPTION
**Pull request by imjacobclark**
Second contribution 👋!

No functional changes here, I simplified the check if we're a `cordovaPlugin`, `javascriptError` or `anything else`.

I also extracted the predicates that checks if the types (null, cordova, js error) to help make the `postMessage` a little easier to read.

Replaced `new Runnable` with a Lambda and the `// Only eval the JS code if this is a valid callback id` comment with a clear variable name.

Moved both public methods to the top of the file and the private methods at the bottom.

### CODE REVIEW

**Comment by dwieeb** [comment]
@imjacobclark Would you mind resolving the conflicts and updating this PR?

**Comment by imjacobclark** [comment]
> @imjacobclark Would you mind resolving the conflicts and updating this PR?

@dwieeb yep, will fix this up tomorrow, thanks for taking a look!

**Comment by imjacobclark** [comment]
@dwieeb conflicts sorted!

File: android/capacitor/src/main/java/com/getcapacitor/MessageHandler.java [hunk 1]
```diff
@@ -35,40 +35,36 @@ public void postMessage(String jsonStr) {
...
         String service = postData.getString("service");
         String action = postData.getString("action");
         String actionArgs = postData.getString("actionArgs");
-        Logger.verbose(Logger.tags("Plugin"), "To native (Cordova plugin): callbackId: " + callbackId + ", service: " + service +
-          ", action: " + action + ", actionArgs: " + actionArgs);
+
+        Logger.verbose(Logger.tags("Plugin"), "To native (Cordova plugin): callbackId: " + callbackId + ", service: " + service + ", action: " + action + ", actionArgs: " + actionArgs);
+
         this.callCordovaPluginMethod(callbackId, service, action, actionArgs);
       } else if (type != null && type.equals("js.error")) {
```
Comment (carlpoole) [inline android/capacitor/src/main/java/com/getcapacitor/MessageHandler.java:52]: Did you mean to use the new `isJavaScriptError` bool here?

Comment (imjacobclark) [reply android/capacitor/src/main/java/com/getcapacitor/MessageHandler.java]: Good spot, fixed!

**Comment by imjacobclark** [comment]
@carlpoole thanks, fixed the comment! What is the process to getting this merged?

**Comment by dwieeb** [comment]
We just merge it. 😉 

Thanks for your contribution, @imjacobclark!

**Pull request merged**